ubuntu-image.rst

```diff
@@ -40,39 +40,39 @@ device for which the image is built.
 
 As part of the model assertion, a `gadget snap`_ is specified.  The gadget
 contains a `gadget.yaml`_ file which contains the exact description of the
-disk image's contents, in YAML format.  The ``gadget.yaml`` file describes
+disk-image contents in the YAML format.  The ``gadget.yaml`` file describes
```

Comment:
I am not sure I understand why "disk-image" is more appropriate than "disk image".

Reply:
I try be consistent in hyphenating [compound adjectives](https://en.wikipedia.org/wiki/Compound_modifier). It's not always necessary, but it can reduce ambiguity in many cases.

I also intentionally reformulated it to remove the possessive, which smacks of anthropomorphism.